src/script/interpreter.h

```diff
@@ -114,32 +120,98 @@ enum
     // Making OP_CODESEPARATOR and FindAndDelete fail any non-segwit scripts
     //
     SCRIPT_VERIFY_CONST_SCRIPTCODE = (1U << 16),
+
+    // Taproot validation
+    //
+    SCRIPT_VERIFY_TAPROOT = (1U << 17),
+
+    // Making unknown taproot leaf versions non-standard
```

Comment:
Unify Taproot capitalization in comments.
```suggestion
    // Making unknown Taproot leaf versions non-standard
```

Reply:
Updated.